index.html

```diff
@@ -46,5 +47,178 @@ <h1>I’m Rashmi.<br> to see<br> you!</h1>
+                                <li><img src="img/Counter.png"></li>
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately<br> 
+                            personalized reads; no accounts or<br> 
+                            sign-ups required.
+                        </p>
+                        <div class="lang">
+                            <ul>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div >
+                             <button class="button-A">See project</button>
+                        </div>
+                    </div>
+      </div>
+                <div class="grid-item"><img src="img/Snapshoot Portfolio (3).png">
+                    <div class="detail">
+                        <h1>Tonic</h1>
+                        <div class="list-01">
+                            <ul>
+                                <li class="canopy">CANOPY</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>Back End Dev</li>
+                                <li><img src="img/Counter.png"></li>
+                                <li>2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately<br> 
+                            personalized reads; no accounts or<br> 
+                            sign-ups required.
+                        </p>
+                        <div class="lang">
+                            <ul>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                                <li>JavaScript</li>
+                            </ul>
+                        </div>
+                        <div >
+                             <button class="button-A">See project</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        
+        <section class="about-myself">
+            <div class="Myself">
+                <section class="welcome">
+                    <h1>About Myself</h1>
+                </section>
+                <div class="info">
+                    <p>I’m a software developer! I can help you<br> 
+                        build a product , feature or website Look<br> 
+                        through some of my work and<br> 
+                        experience! If you like what you see and<br> 
+                        have a project you need coded, don’t<br> 
+                        hesitate to contact me.
+                    </p>
+                </div>
+                <div class="connect">
+                  <p>LET'S CONNECT</p>
```

Comment:
Kindly, I suggest you use an `h2` tag instead of a `p` tag for the sub-heading for good semantics and best practices. 😄